From 52630a9ea5e5f3bdfaf3d5e4f3d3cdc7c112f83e Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Fri, 26 Oct 2018 15:02:56 +0800
Subject: [PATCH] arm64: dts: rockchip: codingstyle for rk3399pro-evb-v10

1. remove duplicated gpu node
2. do not enable uart2
3. reorder for cpu nodes

Change-Id: Ifc11f7b98d4fa3b5cd13464e7454727ac73caebc
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts
index ef0772c94868..b2d6549921e4 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts
@@ -921,10 +921,6 @@
 	status = "okay";
 };
 
-&uart2 {
-	status = "okay";
-};
-
 &usb_host0_ehci {
 	status = "okay";
 };
-- 
2.35.3

